bin/load_images

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+touch .build_built
+for var in "$@"
+do
+    cat /tmp/build_artifacts_$var/.build_built >> .build_built
+    cd /tmp/build_artifacts_$var/docker_images
+    ls -a
+    for file in *; do docker load < "$file"; done
+    rm -rf /tmp/build_artifacts_$var
+    cd -
```

Comment:
I'd avoid "cd -" -- it's stateful and so will be confusing. WHat I usually do is run the thing in a subshell if it needs to change the working directory, e.g. you do:
(
  cd /wqoihfowqihf
  lashfwqjhfwq
)

The ()'s give it a context that goes away at the end.

Or, you don't need to worry about it because the cd in the loop is an absolute path

Or explicitly save $PWD to a variable that you use again later

Reply:
changed to using stored $PWD